apps/web/src/pages/templates/components/TriggerSnippetTabs.tsx

```diff
@@ -85,19 +84,19 @@ export const getCurlTriggerSnippet = (
   snippet?: Record<string, unknown>
 ) => {
   const curlSnippet = `curl --location --request POST '${API_ROOT}/v1/events/trigger' \\
-     --header 'Authorization: ApiKey <REPLACE_WITH_API_KEY>' \\
-     --header 'Content-Type: application/json' \\
-     --data-raw '${JSON.stringify(
-       {
-         name: identifier,
-         to,
-         payload,
-         overrides,
-         ...snippet,
-       },
-       null,
-       2
-     ).replaceAll('\n', '\n       ')}'
+--header 'Authorization: ApiKey <REPLACE_WITH_API_KEY>' \\
```

Comment:
Fixed code indentation for both nodejs and curl

Before 👇🏻

https://github.com/novuhq/novu/assets/39362422/6ef8ff8b-62a4-41e3-9325-048c22a6cd36

After 👇🏻


Uploading Screen Recording 2024-04-12 at 5.09.42 PM.mov…